addons/html_editor/static/src/main/media/icon_plugin.js

```diff
@@ -124,37 +124,37 @@ export class IconPlugin extends Plugin {
         color_apply_overrides: this.applyIconColor.bind(this),
     };
 
-    getSelectedIcon() {
```

Comment:
Same here because the plugin is exported and someone might have extended/overriden it.